policy/evaluator.go

```diff
@@ -322,7 +337,7 @@ type evaluatorActivation struct {
 	memoTerms map[string]ref.Val
 }
 
-func (ctx *evaluatorActivation) ResolveName(name string) (ref.Val, bool) {
+func (ctx *evaluatorActivation) ResolveName(name string) (interface{}, bool) {
```

Comment:
What's going on with changing the return value to interface{}?  interface{} gets really complicated for the caller to handle properly, so it's generally better to return types unless we plan to return an Any since the caller can always assign it to interface{} later.

Reply:
oh, ok, in that case, we probably want to ensure that nil values here are returned using the nil keyword since interface{} will unexpectedly compare non-nil for cases where there is a nil value.

Reply:
It's probably worth mentioning that CEL will handle `nil` as a CEL value of `null`, so the expectation is just that the call returns an appropriate `found` status and the value will be whatever it happens to be.